staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -489,34 +489,34 @@ func (s *serviceCache) allServices() []*v1.Service {
 	return services
 }
 
-func (s *serviceCache) get(serviceName string) (*cachedService, bool) {
+func (s *serviceCache) get(key string) (*cachedService, bool) {
```

Comment:
I would revert this variable name, it just adds noise later when you propose a cherry-pick for this PR

Reply:
Yes, anyway these methods are going to be deleted with `serviceCache`. Updated.